test/fixtures/cookbooks/languages_rust/recipes/default.rb

```diff
@@ -1,8 +1,8 @@
 include_recipe 'chef-sugar::default'
 include_recipe 'languages::default'
 
-package 'curl' unless mac_os_x?
+package 'curl' unless mac_os_x? || windows?
```

Comment:
Makes sense, it's only in the recipe still because I forgot I threw it in there...